frontend/src/app/features/work-packages/components/wp-breadcrumb/wp-breadcrumb.sass

```diff
@@ -5,11 +5,11 @@
   height: initial
 
   &--ellipsed
-    max-width: 420px
+    width: 100%
     @include text-shortener
 
   &--active-parent-select
-    min-width: 320px
+    width: 100%
```

Comment:
This leads to the active select field spanning the whole available width. In the full view of a WP this is way to much

<img width="2021" alt="Bildschirmfoto 2022-01-31 um 10 20 49" src="https://user-images.githubusercontent.com/7457313/151768169-0cebfc29-a68d-4168-b6ce-392e156cbb2f.png">